lesson3/task1.js

```diff
@@ -0,0 +1,16 @@
+//Задание1
+for (i = 10; i >= 0; i--) {
+  if (i % 2 !== 0) {
+    continue;
+  }
+  console.log(i);
+}
```

Comment:
Можем делать проверку и вьіводит необходимое, без лишних действий
```suggestion
for (let i = 10; i >= 0; i--) {
  if (i % 2 === 0) {
  console.log(i);
  }

}
```